cmd/upgrade/upgrade.go

```diff
@@ -53,18 +53,19 @@ func NewCmdVolumeUpgrade(rootCmd *cobra.Command) *cobra.Command {
 		Short:   "Upgrade Volumes, storage engines, and interfaces in openebs application",
 		Aliases: []string{"update"},
 		Run: func(cmd *cobra.Command, args []string) {
-			openebsNs, _ := cmd.Flags().GetString("openebs-namespace")
 			if !util.IsValidCasType(upgrade.CasType) {
 				fmt.Fprintf(os.Stderr, "cas-type %s not supported\n", upgrade.CasType)
 			} else if upgrade.CasType == util.JivaCasType {
-				upgrade.InstantiateJivaUpgrade(openebsNs)
+				upgrade.InstantiateJivaUpgrade()
 			} else {
 				fmt.Println("Only Jiva upgrades are available at this point")
 				fmt.Println("To upgrade other cas-types follow: https://github.com/openebs/upgrade#upgrading-openebs-reources")
 			}
 		},
 	}
 	cmd.SetUsageTemplate(upgradeCmdHelp)
+	// using openebs as default value to namespace
+	cmd.PersistentFlags().StringVarP(&upgrade.OpenebsNs, "openebs-namespace", "", "openebs", "provide openebs-namespace")
```

Comment:
I have used "openebs" as a default namespace, since all the docs are written with this consideration. The more chances are openebs will be the default namespace for all openebs components.

Reply:
set it to `""`, this way one can get to know if `openebs-namespace` has actually been provided via flags or the default value has been used

Reply:
So this is done to shorten the code logic. Default value is "openebs" as all the openebs docs have this as their default namespace value.
If user is not putting any value then "openebs" is used.
The other way, we have to write additional if statement to check if the namespace is provided or not and assign it as "openebs" if not. This is just to get rid of that one if statement and use the default values feature provided by the framework

Reply:
that's right, in the operator YAMLs of many cas-types the namespace has been hardcoded to `"openebs"` but in other cases, when installation is done via helm(which is another interesting project), it's really easy to override it e.g. `helm install ABCD --namespace=random`.